packages/ui/src/components/search-files.tsx

```diff
@@ -12,88 +12,125 @@ import {
   SearchBox,
   Text
 } from '@/components'
+import { debounce } from 'lodash-es'
+
+const markedFileClassName = 'w-full text-foreground-8'
+
+/**
+ * Get marked file component with query
+ * @param file
+ * @param query
+ * @param matchIndex
+ */
+const getMarkedFileElement = (file: string, query: string, matchIndex: number) => {
```

Comment:
Not a blocker, but we should make sure we add return types in the future
```suggestion
const getMarkedFileElement = (file: string, query: string, matchIndex: number): ReactNode => {
```

Also, since this function does return JSX, it's worth considering making it a component.